src/pages/foundation.astro

```diff
@@ -0,0 +1,51 @@
+---
+import Layout from "../layouts/Layout.astro"
+import Heading from "../components/Heading.astro"
+import PostBody from "../components/PostBody.astro"
+
+const title = `DDEV Foundation`
+---
+
+<Layout
+    title={title}
+>
+  <main class="max-w-4xl mx-auto mb-24">
+    <Heading title={`DDEV Foundation`} />
+    <div class="lg:flex">
+      <div class="px-6 lg:px-0 lg:w-2/3">
+        <PostBody>
+          <p>
+            The DDEV Foundation, a Colorado (USA) nonprofit corporation, is the developer of [DDEV](https://ddev.readthedocs.io). It was formerly named the Localdev Foundation. It serves as the fiscal entity for the project and owns the bank account, etc.
+          </p>
+          <p>
+            The DDEV Foundation will be applying for US 501(c)(3) non-profit status in 2023.
+          </p>
+
+          <h2>Where does the money go?</h2>
+
+          <p>
```

Comment:
Might be a good place to link to the about page since this is a little redundant.

Reply:
My basic goal was to move a full statement about the foundation to one place, the content is from localdev.foundation, and the idea is to replace that website. Sometimes a variety of things want a link to the *organization* page for documentation reasons, and they want the answers to their questions on the one page, not as links.

Reply:
Added link to About page though.